### PR TITLE
SV-1364 admin view epao org standards

### PR DESCRIPTION
**Pull request by CodeWolf86**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Api.Types/Models/GetStandardsByOrganisationRequest.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 namespace SFA.DAS.AssessorService.Api.Types.Models
 {
-    public class GetStandardsByOrganisationRequest: IRequest<List<OrganisationStandardSummary>>
+    public class GetAllStandardsByOrganisationRequest: IRequest<List<OrganisationStandardSummary>>
```
Comment (ben1stone) [inline src/SFA.DAS.AssessorService.Api.Types/Models/GetStandardsByOrganisationRequest.cs:9]: Can we rename the file too?

File: src/SFA.DAS.AssessorService.Application/Handlers/ao/GetStandardsByAssessmentOrganisationHandler.cs [hunk 2]
```diff
@@ -10,23 +10,25 @@
 
 namespace SFA.DAS.AssessorService.Application.Handlers.ao
 {
-    public class GetStandardsByAssessmentOrganisationHandler: IRequestHandler<GetStandardsByOrganisationRequest, List<OrganisationStandardSummary>>
+    /// <summary>
+    /// This handler doesn't restrict based on efffective from / to and status of live
+    /// As it's called from the Admin Service and the Admin side needs to know about active and withdrawn standards.
+    /// </summary>
+    public class GetStandardsByAssessmentOrganisationHandler: IRequestHandler<GetAllStandardsByOrganisationRequest, List<OrganisationStandardSummary>>
```
Comment (ben1stone) [inline src/SFA.DAS.AssessorService.Application/Handlers/ao/GetStandardsByAssessmentOrganisationHandler.cs:17]: Can we update handler name to match with the request and service method?

**Review by ben1stone — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Api.Types/Models/GetStandardsByOrganisationRequest.cs (hunk 1 above)
Comment (CodeWolf86) [reply src/SFA.DAS.AssessorService.Api.Types/Models/GetStandardsByOrganisationRequest.cs:7]: Have renamed 👍

File: src/SFA.DAS.AssessorService.Application/Handlers/ao/GetStandardsByAssessmentOrganisationHandler.cs (hunk 2 above)
Comment (CodeWolf86) [reply src/SFA.DAS.AssessorService.Application/Handlers/ao/GetStandardsByAssessmentOrganisationHandler.cs]: Have renamed 👍

File: src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs [hunk 3]
```diff
@@ -217,14 +219,34 @@ [dbo].[OrganisationStandardVersion] osv
...
+                ORDER BY s.VersionMajor, s.VersionMinor";
+
+            using (var multi = await _unitOfWork.Connection.QueryMultipleAsync(sql, new { organisationStandardId }))
+            {
+                orgStandard = await multi.ReadSingleAsync<OrganisationStandard>();
+                var standardVersions = multi.Read<OrganisationStandardVersion>();
 
-            return await _unitOfWork.Connection.QuerySingleAsync<OrganisationStandard>(sql, new { organisationStandardId });
+                orgStandard.Versions = standardVersions?.ToList();
+            }
```
[the remark this answers is not in the document]
Comment (CodeWolf86) [reply src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs:243]: Will merge then do directly on 1377 to unblock Uma and his testing.

**Review by ben1stone — APPROVED** [review]

**Pull request merged**